ilp-core/src/main/java/org/interledger/InterledgerPacket.java

```diff
@@ -39,7 +38,7 @@
      * A handler for allowing callers to specify logic based upon an unknown result type. This class
      * can be used in the following manner:
      *
-     * <pre>
+     * <p><pre>
```

Comment:
I get a checkstyle build failure with this -- does this build properly on your local machine?